src/sentry/apidocs/hooks.py

```diff
@@ -132,11 +135,23 @@ def __write_ownership_data(ownership_data: Dict[ApiOwner, Dict]):
     file_to_write.close()
 
 
+class CustomEndpointEnumerator(EndpointEnumerator):
+    _non_capturing = re.compile(r"\(\?:[a-z-]+\|[a-z-]+\)")
+
+    def get_path_from_regex(self, path_regex: str) -> str:
+        # django 4.x changes to simplify_regex breaks these urls
+        path_regex = self._non_capturing.sub("{var}", path_regex)
+        return super().get_path_from_regex(path_regex)
```

Comment:
Do this need to include these small [bugfix lines](https://github.com/tfranzel/drf-spectacular/blob/f31238e176c06fde20a859d591e3c5661576d44d/drf_spectacular/generators.py#L44-L45)? I'm not too sure what it does but just wanna double check.

Reply:
https://docs.python.org/3/library/functions.html#super

Reply:
oh right whoops